projects/packages/connection/src/class-rest-connector.php

```diff
@@ -142,6 +142,27 @@ public function __construct( Manager $connection ) {
 				),
 			)
 		);
+
+		// Get authorization URL.
+		register_rest_route(
+			'jetpack/v4',
+			'/connection/authorize_url',
+			array(
+				'methods'             => WP_REST_Server::READABLE,
+				'callback'            => array( $this, 'connection_authorize_url' ),
+				'permission_callback' => '__return_true',
```

Comment:
Shouldn't we limit this to folks who have the `jetpack_connect` cap?

Reply:
Fixed, good catch!
https://github.com/Automattic/jetpack/pull/19837/commits/3fc4dcafbdb147e14e3d8f034f2aa04cdba75364